src/main/java/org/cbioportal/legacy/web/PublicVirtualStudiesController.java

```diff
@@ -113,13 +153,28 @@ private void unPublishVirtualStudy(String id) {
           "The virtual study with id=" + id + " has not been found in the public list.");
     }
     VirtualStudyData virtualStudyData = virtualStudyToUnPublish.getData();
+    checkIfVSWasPublished(id, virtualStudyData);
+    virtualStudyData.setUsers(Set.of(virtualStudyData.getOwner()));
+    sessionServiceRequestHandler.updateVirtualStudy(virtualStudyToUnPublish);
```

Comment:
what happens when I try to publish a VS that has been unpublished? Will it fail because the ID is the same, will it just overwrite it? If the latter is the case, we might not really need `dropPublicVirtualStudy`.